test/client/integration.test.ts

```diff
@@ -45,24 +54,88 @@ describe("Client Integration", function() {
                 new Pubkey(user.pubkey)
             );
             await storagePacker.state.update(user.stateID, state);
-
-            // Setup syncer L2 state
-            // Replace with L1 deposits once implemented
-            await storageSyncer.state.update(user.stateID, state);
         }
         await storagePacker.pubkey.commit();
         await storagePacker.state.commit();
-        await storageSyncer.state.commit();
 
         const parameters = PRODUCTION_PARAMS;
         parameters.USE_BURN_AUCTION = false;
-        parameters.GENESIS_STATE_ROOT = storagePacker.state.root;
+        // TODO Check that this change is correct and makes sense.
+        parameters.GENESIS_STATE_ROOT = storageSyncer.state.root;
```

Comment:
I believe this change is correct. Until we implement genesis sync, this start state should be the syncer, not the packer.

Reply:
It depends on what we want to have in the genesis state tree.
Before we have a functioning deposit feature, we are not able to introduce a new state leaf into the state tree after the genesis. That's why we created 32 state leaves in the genesis state tree.

Since both the syncer and the packer have these 32 leaves, so it doesn't matter we use storagePacker or storageSyncer here. They should have the same state tree root.

I see you removed the setup of storageSyncer. But that would break something because now syncer and the packer has different genesis state.

Reply:
Reverted, will discuss how we want to do this offline with @ChihChengLiang